docs/commands/save.md

```diff
@@ -0,0 +1,28 @@
+# save
+
+This command saves the contents of the pipeline to a file. Use this in combination with the `to-json`, `to-csv`, ... commands to save the contents in the specified format.
+
+Syntax: `save (path) {flags}`
+
+### Parameters:
+
+* `(path)` the path to save contents to
+
+### Flags
+
+    --raw
+      treat values as-is rather than auto-converting based on file extension
+
+## Example
+
+You can save the name of files in a directory like this:
+
+```shell
+> ls | where type == File | pick name | save
```

Comment:
This example errors for me. I think you can only use `save` without a filename if you used `open` earlier in the pipeline so that the values know where they came from.